api/Dockerfile.redhat-ubi

```diff
@@ -14,8 +14,11 @@ USER root
 RUN yum -y update-minimal --security --sec-severity=Important --sec-severity=Critical
 USER 1001
 
-ADD --chown=1001:0 requirements.txt .
-RUN pip install -r requirements.txt
+ENV POETRY_VERSION=1.5.0
```

Comment:
Do we need this, since we've defined a default version in the Makefile?

Reply:
I will update this Dockerfile, after the current Dockerfile is reviewed and approved.